src/Command/MigrateCommand.php

```diff
@@ -282,15 +457,21 @@ private function createNewDirectory($path)
     }
 
     /**
-     * param string $path
+     * @param string $path
+     * @param bool $disableConfigProcessorInjection
      */
-    private function injectBridgeModule($path, SymfonyStyle $io)
+    private function injectBridgeModule($path, $disableConfigProcessorInjection, SymfonyStyle $io)
```

Comment:
yday I've added support for Symfony Console 2.1+ but ... we need to check if it works with SymfonyStyle... I haven't checked it, tbh

Reply:
They were added in 2.7. As such, that needs to be our minimum version.

I think that's fairly reasonable, as it's 4.5 years old.